mint/create-data-files.sh

```diff
@@ -15,7 +15,8 @@
 #  limitations under the License.
 #
 
-MINT_DATA_DIR="$MINT_ROOT_DIR/data"
+# MINT_DATA_DIR="$MINT_ROOT_DIR/data"
+MINT_DATA_DIR=${MINT_DATA_DIR:-/mint/data}
```

Comment:
Allows setting a custom location, not a big requirement but more bash friendly implementation.

Reply:
@ebozduman I discussed this with @harshavardhana . Since `MINT_ROOT_DIR` itself is already configurable, this change is not required and the previous code (`MINT_DATA_DIR="$MINT_ROOT_DIR/data"`) can be retained.